x-pack/plugins/spaces/public/nav_control/nav_control_popover.tsx

```diff
@@ -105,7 +109,7 @@ export class NavControlPopover extends Component<Props, State> {
         anchorPosition={this.props.anchorPosition}
         panelPaddingSize="none"
         repositionOnScroll
-        ownFocus
+        ownFocus={true}
```

Comment:
nit: Truthy boolean props don't need a value (See `repositionOnScroll`)

Reply:
I think this was an artifact from when I was testing is as false. I'll update it.